drivers/vmwarevsphere/flags.go

```diff
@@ -162,6 +162,16 @@ func (d *Driver) GetCreateFlags() []mcnflag.Flag {
 			Usage:  "If using a non-B2D image the uploaded keys will need chown'ed, defaults to staff e.g. docker:staff",
 			Value:  defaultSSHUserGroup,
 		},
+		mcnflag.StringSliceFlag{
```

Comment:
you sure we don't want an env var here?

Reply:
Yup. We are pushing for this to be ran through rancher only so it's really not needed.